recipes/cssselect/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "cssselect" %}
+{% set version = "0.9.2" %}
+{% set sha256 = "76f090e40297b21fb57459b0ea73c8e46046feaf872909224d1fb861c1703ef5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: cssselect-{{ version }}.tar.gz
+  url: https://github.com/scrapy/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
Add the `name` seems too much here. Is it alright if we just make it `cssselect`.